code/mob/living/carbon/human/procs/Life.dm

```diff
@@ -934,19 +934,18 @@
 				for (var/datum/gas/rad_particles/RV in breath.trace_gases)
 					src.changeStatus("radiation", RV.moles, 2 SECONDS)
 
-		if (breath.temperature > (T0C+66) && !src.is_heat_resistant()) // Hot air hurts :(
-			if (!has_cyberlungs || (has_cyberlungs && (breath.temperature > (T0C+500))))
-				var/burn_damage = min((breath.temperature - (T0C+66)) / 3,10) + 6
-				TakeDamage("chest", 0, burn_damage, 0, DAMAGE_BURN)
-				if (prob(20))
-					boutput(src, "<span class='alert'>You feel a searing heat in your lungs!</span>")
-					if (src.organHolder)
-						src.organHolder.damage_organs(0, max(burn_damage, 3), 0, list("left_lung", "right_lung"), 80)
-
-				hud.update_fire_indicator(1)
-				if (prob(4))
-					boutput(src, "<span class='alert'>Your lungs hurt like hell! This can't be good!</span>")
-					//src.contract_disease(new/datum/ailment/disability/cough, 1, 0) // cogwerks ailment project - lung damage from fire
+		if (breath.temperature > min(organHolder.left_lung.temp_tolerance, organholder.right_lung.temp_tolerance) && !src.is_heat_resistant()) // Hot air hurts :(
```

Comment:
```suggestion
		if (breath.temperature > min(organHolder.left_lung.temp_tolerance, organHolder.right_lung.temp_tolerance) && !src.is_heat_resistant()) // Hot air hurts :(
```

Reply:
it was identical to what I pushed, so I didn't commit it.